core/templates/dev/head/editor/exploration_settings.html

```diff
@@ -431,7 +431,7 @@
 
     <h3>
       Initial parameter changes
-      <span class="glyphicon glyphicon-info-sign" tooltip="These changes are applied before the learner begins the exploration." tooltip-placement="right" style="font-size: 12px; padding-left: 18px;"></span>
+      <i class="material-icons"tooltip="These changes are applied before the learner begins the exploration." tooltip-placement="right" style="font-size: 12px; padding-left: 18px;">&#xE88E;</i>
```

Comment:
This icon is pretty much unreadable. Can it be bigger? Maybe remove the font size and add md-18.